component/api/src/main/java/org/exoplatform/social/metadata/MetadataService.java

```diff
@@ -172,7 +172,7 @@ MetadataItem createMetadataItem(MetadataObject metadataObject,
    *                 and an objectId (the object technical identifier. In
    *                 general we use here the DB identifier of the object).
    */
-  void deleteMetadataItemsByObject(MetadataObject object);
+  void deleteMetadataItemsByObject(MetadataObject object, boolean broadcast);
```

Comment:
Please avoid changing the existing API and broadcast event all time.

```suggestion
  void deleteMetadataItemsByObject(MetadataObject object);
```